x-pack/test/functional_with_es_ssl/apps/cases/list_view.ts

```diff
@@ -18,104 +19,161 @@ export default ({ getPageObject, getService }: FtrProviderContext) => {
 
   describe('cases list', () => {
     before(async () => {
-      await common.navigateToApp('cases');
-      await cases.api.deleteAllCases();
+      await cases.navigation.navigateToApp();
     });
 
     after(async () => {
       await cases.api.deleteAllCases();
     });
 
-    beforeEach(async () => {
-      await common.navigateToApp('cases');
-    });
-
-    it('displays an empty list with an add button correctly', async () => {
-      await testSubjects.existOrFail('cases-table-add-case');
+    describe('empty state', () => {
+      it('displays an empty list with an add button correctly', async () => {
+        await testSubjects.existOrFail('cases-table-add-case');
+      });
     });
 
-    it('lists cases correctly', async () => {
+    describe('listing', () => {
```

Comment:
nit: i don't see why transform this from an `it` to a whole suite.

Reply:
I want some dedicated before/after setups only for this test.

Reply:
I understand. But keep in mind is perfectly ok to have the "before" and "after" code in standalone test. `before` and `after` are merely functions called before calling the test function. If the suite has a single test this split is not necessary.

I won't get picky for this, just making you aware that it is not necessary to write it this way.

Reply:
You can put a `before`/`after` inside an `it`?

Reply:
more like you can simply put the code that would be inside `before`/`after` inside the `it`.

e.g.:

```
describe("test suite", () => {
   before( async ()=> {
      await setupThis();
   });
  after( async ()=> {
      await sunsetThat();
   });
   it("the test", async ()=>{
     await testMe();
   });
});

```

can be rewritten to this less boilerplate version when you know the suite won't grow:


```
   it("the test", async ()=>{
	  await setupThis();
      await testMe();
      await sunsetThat();
   });
```

Reply:
oh, I see what you mean. Ok, that makes sense 🙂. I will keep that in mind for the upcoming PRs.